Dockerfile.prod

```diff
@@ -0,0 +1,20 @@
+FROM elixir:1.13-alpine
+
+# Install build dependencies
+RUN apk add --no-cache build-base git python3
+
+# Install frontend dependencies
+RUN apk add --no-cache nodejs npm
+RUN npm install -g npx
+
+# Install imagemagick dependencies
+RUN apk add --no-cache file imagemagick
+
+# Install hex + rebar
+RUN mix local.hex --force && \
+    mix local.rebar --force
+
+
+WORKDIR /app
+
+CMD [ "sh", "-c", "mix deps.get; mix compile; mix assets.deploy; mix phx.server" ]
```

Comment:
This Dockerfile should be splitted in two sections, build the release, and then copy the files to a new container.

You should just use the phoenix generated Dockerfile, `mix phx.gen.release --docker`.

Learn more [here](https://hexdocs.pm/phoenix/Mix.Tasks.Phx.Gen.Release.html).